website/src/app/views/our-team/members.js

```diff
@@ -239,6 +239,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/subin-kim-bba754180/',
         imageURL: subin,
       },
+      {
```

Comment:
alphabetical order (sr < su)

Reply:
Should be fixed now unless I forgot the alphabet again.